voltixApp/VoltixApp/Services/Thorchain/ThorchainBroadcastTransactionService.swift

```diff
@@ -0,0 +1,73 @@
+    //
+    //  ThorchainBroadcastTransactionService.swift
+    //  VoltixApp
+    //
+    //  Created by Enrique Souza Soares on 07/03/2024.
+    //
+
+import Foundation
+
+extension ThorchainService {
+    
+    func broadcastTransaction(jsonString: String, completion: @escaping (Result<String, Error>) -> Void) {
+        let url = URL(string: "https://thornode.ninerealms.com/cosmos/tx/v1beta1/txs")!
```

Comment:
Use the Endpoint file to store the URL string.
The same goes for the URL string used for the fetchAccountNumber call.

Reply:
Done.